File: javaTool/src/main/java/nl/um/cds/triplifier/rdf/AnnotationFactory.java
package nl.um.cds.triplifier.rdf;

import nl.um.cds.triplifier.rdf.ontology.DBO;

import java.util.Properties;

public class AnnotationFactory extends RdfFactory{

    public AnnotationFactory(Properties props) {
        super(props);
        this.initialize();
    }

    private void initialize() {
        this.initializeRdfStore();
        this.context = vf.createIRI("http://annotation.local/");
    }
}

File: pythonTool/rdf/annotation_factory.py
from typing import Dict
from rdflib import URIRef

from .rdf_factory import RdfFactory


class AnnotationFactory(RdfFactory):
    def __init__(self, props: Dict[str, str]):
        super().__init__(props)
        self.initialize()

    def initialize(self) -> None:
        self.initialize_rdf_store()
        self.context = URIRef("http://annotation.local/")
